tests/common.go

```diff
@@ -1228,6 +1234,41 @@ func ValidateMountOptionsWithPureVolumes(ctx *scheduler.Context, errChan ...*cha
 
 }
 
+// ValidateCreateOptionsWithPureVolumes is the ginkgo spec for executing a check for Create options in a volume
+func ValidateCreateOptionsWithPureVolumes(ctx *scheduler.Context, errChan ...*chan error) {
+	var FSType1 = "xfs"
+	var FSType2 = "ext4"
+	var requiredXfsCreateoption = "-b 2048"
+	var requiredExt4Createoption = "-b size=2048"
+	vols, err := Inst().S.GetVolumes(ctx)
+	log.FailOnError(err, "Failed to get app %s's volumes", ctx.App.Key)
+	log.Infof("volumes of app %s are %s", ctx.App.Key, vols)
+	for _, v := range vols {
+		pvcObj, err := k8sCore.GetPersistentVolumeClaim(v.Name, v.Namespace)
+		if err != nil {
```

Comment:
No need for nil check. The failonError seems to take care of it implicitly.

Reply:
Done